projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -62,6 +72,54 @@ public function get_data() {
 			'checklist_statuses' => get_option( 'launchpad_checklist_tasks_statuses', array() ),
 		);
 	}
+
+	/**
+	 * Updates Launchpad-related options and returns the result
+	 *
+	 * @param array $request The unsanitized request values.
+	 *
+	 * @return array Associative array with updated site options
+	 */
+	public function update_site_options( $request ) {
+		$updated = array();
+		$input   = $request->get_json_params();
+
+		foreach ( $input as $key => $value ) {
+			if ( ! is_array( $value ) ) {
+				$value = trim( $value );
+			}
+
+			switch ( $key ) {
+				case 'launchpad_checklist_tasks_statuses':
+					$launchpad_checklist_tasks_statuses_option = get_option( 'launchpad_checklist_tasks_statuses' );
+
+					$filtered_input_array = array_filter(
+						(array) $value,
+						function ( $array_value ) {
+							return is_bool( $array_value );
+						},
+						ARRAY_FILTER_USE_BOTH
```

Comment:
Given we're not using key and value in the callback, may not need ARRAY_FILTER_USE_BOTH?